### PR TITLE
inardini - experiments cuj3 notebook release

### PR DESCRIPTION
**Pull request by inardini**
A PR to release the official experiments notebook - CUJ3.

If you are opening a PR for `Official Notebooks` under the [notebooks/official](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/notebooks/official) folder, follow this mandatory checklist:
- [x] Use the [notebook template](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/notebook_template.ipynb) as a starting point.
- [x] Follow the style and grammar rules outlined in the above notebook template.
- [x] Verify the notebook runs successfully in Colab since the automated tests cannot guarantee this even when it passes.
- [x] Passes all the required automated checks. You can locally test for formatting and linting with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/CONTRIBUTING.md#code-quality-checks).
- [x] You have consulted with a tech writer to see if tech writer review is necessary. If so, the notebook has been reviewed by a tech writer, and they have approved it.
- [x] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/official/CODEOWNERS) file under the `Official Notebooks` section, pointing to the author or the author's team.
- [x] The Jupyter notebook cleans up any artifacts it has created (datasets, ML models, endpoints, etc) so as not to eat up unnecessary resources.

### CODE REVIEW

File: notebooks/official/experiments/comparing_pipeline_runs.ipynb [hunk 1]
```diff
@@ -0,0 +1,1045 @@
+{
```
Comment (andrewferlitsch) [inline notebooks/official/experiments/comparing_pipeline_runs.ipynb:1]: <p>move pip install into above cell (single cell for installs)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/638/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Comment (andrewferlitsch) [inline notebooks/official/experiments/comparing_pipeline_runs.ipynb:1]: <p>Seems to be missing demonstrating comparing the 5 runs?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/638/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='53'/>

Comment (inardini) [reply notebooks/official/experiments/comparing_pipeline_runs.ipynb:1]: done

[the remark this answers is not in the document]
Comment (inardini) [reply notebooks/official/experiments/comparing_pipeline_runs.ipynb:1]: done

[the remark this answers is not in the document]
Comment (inardini) [reply notebooks/official/experiments/comparing_pipeline_runs.ipynb:1]: thanks andy. I'll do that

**Review by andrewferlitsch — APPROVED** [review]

**Comment by andrewferlitsch** [comment]
Failed on import:

     367 try:
Step #3: --> 368     import pandas as pd
Step #3:     369 except ImportError:
Step #3: 
Step #3: File ~/.local/lib/python3.9/site-packages/pandas/__init__.py:22, in <module>
Step #3:      21 # numpy compat
Step #3: ---> 22 from pandas.compat import is_numpy_dev as _is_numpy_dev
Step #3:      24 try:
Step #3: 
Step #3: File ~/.local/lib/python3.9/site-packages/pandas/compat/__init__.py:15, in <module>
Step #3:      14 from pandas._typing import F
Step #3: ---> 15 from pandas.compat.numpy import (
Step #3:      16     is_numpy_dev,
Step #3:      17     np_version_under1p19,
Step #3:      18     np_version_under1p20,
Step #3:      19 )
Step #3:      20 from pandas.compat.pyarrow import (
Step #3:      21     pa_version_under1p01,
Step #3:      22     pa_version_under2p0,
Step #3:      23     pa_version_under3p0,
Step #3:      24     pa_version_under4p0,
Step #3:      25 )
Step #3: 
Step #3: File ~/.local/lib/python3.9/site-packages/pandas/compat/numpy/__init__.py:4, in <module>
Step #3:       2 import numpy as np
Step #3: ----> 4 from pandas.util.version import Version
Step #3:       6 # numpy versioning
Step #3: 
Step #3: File ~/.local/lib/python3.9/site-packages/pandas/util/__init__.py:1, in <module>
Step #3: ----> 1 from pandas.util._decorators import (  # noqa:F401
Step #3:       2     Appender,
Step #3:       3     Substitution,
Step #3:       4     cache_readonly,
Step #3:       5 )
Step #3:       7 from pandas.core.util.hashing import (  # noqa:F401
Step #3:       8     hash_array,
Step #3:       9     hash_pandas_object,
Step #3:      10 )
Step #3: 
Step #3: File ~/.local/lib/python3.9/site-packages/pandas/util/_decorators.py:14, in <module>
Step #3:      12 import warnings
Step #3: ---> 14 from pandas._libs.properties import cache_readonly  # noqa:F401
Step #3:      15 from pandas._typing import F
Step #3: 
Step #3: File ~/.local/lib/python3.9/site-packages/pandas/_libs/__init__.py:14, in <module>
Step #3:      13 from pandas._libs.interval import Interval
Step #3: ---> 14 from pandas._libs.tslibs import (
Step #3:      15     NaT,
Step #3:      16     NaTType,
Step #3:      17     OutOfBoundsDatetime,
Step #3:      18     Period,
Step #3:      19     Timedelta,
Step #3:      20     Timestamp,
Step #3:      21     iNaT,
Step #3:      22 )
Step #3: 
Step #3: File ~/.local/lib/python3.9/site-packages/pandas/_libs/tslibs/__init__.py:30, in <module>
Step #3:      29 from pandas._libs.tslibs import dtypes
Step #3: ---> 30 from pandas._libs.tslibs.conversion import (
Step #3:      31     OutOfBoundsTimedelta,
Step #3:      32     localize_pydatetime,
Step #3:      33 )
Step #3:      34 from pandas._libs.tslibs.dtypes import Resolution
Step #3: 
Step #3: File ~/.local/lib/python3.9/site-packages/pandas/_libs/tslibs/conversion.pyx:63, in init pandas._libs.tslibs.conversion()
Step #3: 
Step #3: File ~/.local/lib/python3.9/site-packages/pandas/_libs/tslibs/parsing.pyx:1, in init pandas._libs.tslibs.parsing()
Step #3: 
Step #3: File ~/.local/lib/python3.9/site-packages/pandas/_libs/tslibs/offsets.pyx:21, in init pandas._libs.tslibs.offsets()
Step #3: 
Step #3: ModuleNotFoundError: No module named 'dateutil.easter'
Step #3: 
Step #3: During handling of the above exception, another exception occurred:
Step #3: 
Step #3: ImportError                               Traceback (most recent call last)
Step #3: Input In [24], in <cell line: 2>()
Step #3:       1 # see state of all pipelineJob
Step #3: ----> 2 vertex_ai.get_experiment_df(EXPERIMENT_NAME)
Step #3: 
Step #3: File /usr/local/lib/python3.9/site-packages/google/cloud/aiplatform/metadata/metadata.py:437, in _ExperimentTracker.get_experiment_df(self, experiment)
Step #3:     434 else:
Step #3:     435     experiment = experiment_resources.Experiment(experiment)
Step #3: --> 437 return experiment.get_data_frame()
Step #3: 
Step #3: File /usr/local/lib/python3.9/site-packages/google/cloud/aiplatform/metadata/experiment_resources.py:370, in Experiment.get_data_frame(self)
Step #3:     368     import pandas as pd
Step #3:     369 except ImportError:
Step #3: --> 370     raise ImportError(
Step #3:     371         "Pandas is not installed and is required to get dataframe as the return format. "
Step #3:     372         'Please install the SDK using "pip install python-aiplatform[metadata]"'
Step #3:     373     )
Step #3:     375 service_request_args = dict(
Step #3:     376     project=self._metadata_context.project,
Step #3:     377     location=self._metadata_context.location,
Step #3:     378     credentials=self._metadata_context.credentials,
Step #3:     379 )
Step #3:     381 filter_str = metadata_utils._make_filter_string(
Step #3:     382     schema_title=sorted(
Step #3:     383         list(_SUPPORTED_LOGGABLE_RESOURCES[context._Context].keys())
Step #3:     384     ),
Step #3:     385     parent_contexts=[self._metadata_context.resource_name],
Step #3:     386 )
Step #3: 
Step #3: ImportError: Pandas is not installed and is required to get dataframe as the return format. Please install the SDK using "pip install python-aiplatform[metadata]"
Step #3: 
Finished Step #3
ERROR
ERROR: build step 3 "gcr.io/cloud-devrel-public-resources/python-samples-testing-docker:latest" failed: step exited with non-zero status: 1

**Comment by kweinmeister** [comment]
The error suggests that you use `pip install python-aiplatform[metadata]`. It looks like you are currently using `git+https://github.com/sasha-gitg/python-aiplatform@main`.

**Comment by andrewferlitsch** [comment]
Correct myself. Of the 6 notebooks ran, this one failed:

#3: ImportError                               Traceback (most recent call last)
Step #3: Input In [24], in <cell line: 2>()
Step #3:       1 # see state of all pipelineJob
Step #3: ----> 2 vertex_ai.get_experiment_df(EXPERIMENT_NAME)
Step #3: 
Step #3: File /usr/local/lib/python3.9/site-packages/google/cloud/aiplatform/metadata/metadata.py:437, in _ExperimentTracker.get_experiment_df(self, experiment)
Step #3:     434 else:
Step #3:     435     experiment = experiment_resources.Experiment(experiment)
Step #3: --> 437 return experiment.get_data_frame()
Step #3: 
Step #3: File /usr/local/lib/python3.9/site-packages/google/cloud/aiplatform/metadata/experiment_resources.py:370, in Experiment.get_data_frame(self)
Step #3:     368     import pandas as pd
Step #3:     369 except ImportError:
Step #3: --> 370     raise ImportError(
Step #3:     371         "Pandas is not installed and is required to get dataframe as the return format. "
Step #3:     372         'Please install the SDK using "pip install python-aiplatform[metadata]"'
Step #3:     373     )
Step #3:     375 service_request_args = dict(
Step #3:     376     project=self._metadata_context.project,
Step #3:     377     location=self._metadata_context.location,
Step #3:     378     credentials=self._metadata_context.credentials,
Step #3:     379 )
Step #3:     381 filter_str = metadata_utils._make_filter_string(
Step #3:     382     schema_title=sorted(
Step #3:     383         list(_SUPPORTED_LOGGABLE_RESOURCES[context._Context].keys())
Step #3:     384     ),
Step #3:     385     parent_contexts=[self._metadata_context.resource_name],
Step #3:     386 )
Step #3: 
Step #3: ImportError: Pandas is not installed and is required to get dataframe as the return format. Please install the SDK using "pip install python-aiplatform[metadata]"
Step #3: 
Finished Step #3

**Comment by ivanmkc** [comment]
LGTM, merging as the `vertex-ai-notebook-execution-test` finished running successfully.

**Pull request merged**